BUILD.adoc

```diff
@@ -537,6 +540,11 @@ recent version.
     included in .gitattributes as well.
 ====
 
+Node.js is best obtained through `nvm`.
```

Comment:
I don't think it's that important to give guidance on how to obtain npm - it's easy enough to search online, and pretty much any npm distribution should work fine. But I don't feel strongly.

Reply:
Depends on the audience.
Personally, I prefer `BUILD.md`s to be written in such a way that a braindead person could build the project in 1 minute...

Reply:
That's fair, it's always nice when all I have to do is follow some instructions without googling anything.